doc/book/box/data_model.rst

```diff
@@ -554,6 +554,104 @@ Charts explaining the precise differences from DUCET order are
 in the
 `Common Language Data Repository <https://unicode.org/cldr/charts/30/collation>`_.
 
+.. _index-box_foreign_keys:
+
+------------
+Foreign keys
+------------
+
+**Foreign keys** provide links between related spaces, therefore maintaining the
+`referential integrity <https://en.wikipedia.org/wiki/Referential_integrity>`_
+of the database.
+
+Some fields can only contain values present in other spaces. For example,
+shop orders always belong to existing customers. Hence, all values of the ``customer``
+field of the ``orders`` space must exist in the ``customers`` space. In this case,
+``customers`` is a **parent space** for ``orders`` (its **child space**). Foreign keys
+enable automatic checks of values presence in the parent space upon each insertion and
+modification of the child space.
+
+*****************
+Foreign key types
+*****************
+
+There are two types of foreign keys in Tarantool:
+
+* *field foreign keys* check that the value being assigned to a field
+  is present in a particular field of another space. For example, the ``customer``
+  value of an ``orders`` tuple must match an ``id`` stored in the ``customers`` space.
+
+* *tuple foreign keys* check that multiple fields of a tuple have a match in
+  another space. For example, if the ``orders`` space has fields ``customer_id``
+  and ``customer_name``, a tuple foreign key can check that the ``customers`` space
+  contains a tuple with both these values in the corresponding fields.
```

Comment:
```suggestion
* *Field foreign keys* check that the value being assigned to a field
  is present in a particular field of another space. For example, the ``customer``
  value in a tuple from the ``orders`` space must match an ``id`` stored in the ``customers`` space.

* *Tuple foreign keys* check that multiple fields of a tuple have a match in
  another space. For example, if the ``orders`` space has fields ``customer_id``
  and ``customer_name``, a tuple foreign key can check that the ``customers`` space
  contains a tuple with both these values in the corresponding fields.
```
Judging from the previous example, ``orders`` is a space. It is not entirely clear.
List punctuation: https://www.tarantool.io/en/doc/latest/contributing/docs/style/#lists